app/src/layouts/tabular/tabular.vue

```diff
@@ -112,7 +113,7 @@ function removeField(fieldKey: string) {
 <template>
 	<div class="layout-tabular">
 		<v-table
-			v-if="loading || (itemCount && itemCount > 0 && !error)"
+			v-if="loading || ((items.length ?? 0) > 0 && !error)"
```

Comment:
Same note about assertion

```suggestion
			v-if="loading || (items.length > 0 && !error)"
```